templates/jquery/js/custom-templates/er-dashboard/files/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>$(description)</title>
+	<link href="./styles.css" rel="stylesheet" />
+	$(cssBlock)
+	$(scriptBlock)
+
+	<script src="./scripts/dataTiles.js"></script>
+    <script src="./scripts/dataPatients.js"></script>
+    <script src="./scripts/dataOrders.js"></script>
+    <script src="./scripts/vitals.js"></script>
+
+    <script src="./scripts/scripts.js"></script>
+
+</head>
+<body onload="self.setInterval('setDateTime()', 1000);">
```

Comment:
Since the scripts already wait on `jQuery.ready` to initialize the view  (tile manager, etc), can this and the `createGrid();` below be tied to a single entry point?